### PR TITLE
create story template

### PR DESCRIPTION
**Pull request by aresnik11**
### Overview

Creates a template mdx and story file based on the new template https://www.figma.com/design/ReGfRNillGABAj5SlITalN/%F0%9F%93%90-Gamut?node-id=29940-42348&node-type=canvas&t=oZQXLsQmthuXYAnS-0

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [GM-838]
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

[GM-838]: https://skillsoftdev.atlassian.net/browse/GM-838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/styleguide/src/lib/Template/template.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { Canvas, Controls, Meta, Story } from '@storybook/blocks';
+
```
Comment (dreamwasp) [inline packages/styleguide/src/lib/Template/template.mdx:2]: i wonder since we have a baseUrl if we can make this a relative path 🤔

Comment (dreamwasp) [reply packages/styleguide/src/lib/Template/template.mdx:2]: nvm, theres a few more thing we need to do to set this up, i'll get it working in my layout branch

Comment (dreamwasp) [reply packages/styleguide/src/lib/Template/template.mdx:2]: https://github.com/Codecademy/gamut/pull/2961/commits/1d729c443b4c45a7d618ad480877a39e539c8b0e

**Review by jakemhiller — APPROVED** [review]
lgtm!

**Review by dreamwasp — APPROVED** [review]

**Pull request merged**